api/trace/testtrace/span.go

```diff
@@ -69,6 +70,30 @@ func (s *Span) End(opts ...trace.EndOption) {
 	s.ended = true
 }
 
+func (s *Span) RecordError(ctx context.Context, err error, opts ...trace.ErrorOption) {
+	if err == nil || s.ended {
+		return
+	}
+
+	cfg := trace.ErrorConfig{}
```

Comment:
I just think that we can initialize default config for ErrorConfig. For example:

```
cfg := trace.ErrorConfig{
  Timestamp: time.Now(),
  Status:  codes.OK,
}
```

After that, it'll replace by for range opts automatically. No needs to check each field in ErrorConfig.

Reply:
`codes.OK` is the zero value, so it doesn't need to be specified.  `Timestamp` would still need to be checked for `cfg.Timestamp.IsZero()` in case the user provided a zero-valued timestamp, so I think it best to leave it initialized to the zero value and explicitly handle it.

Reply:
Ah ok.